### ISSUE TITLE
«Front controller reached 100 router match iterations» in Magento 2.2.2 because the base router ordering has been changed from 20 to 30

### ISSUE THREAD

**Issue by dmitrii-fediuk** [mage2pro/cms#1]
```
Exception #0 (LogicException): Front controller reached 100 router match iterations
#0 lib/internal/Magento/Framework/Interception/Interceptor.php(58): Magento\Framework\App\FrontController->dispatch(Object(Magento\Framework\App\Request\Http))
#1 lib/internal/Magento/Framework/Interception/Interceptor.php(138): Magento\Framework\App\FrontController\Interceptor->___callParent('dispatch', Array)
#2 app/code/Magento/Store/App/FrontController/Plugin/RequestPreprocessor.php(94): Magento\Framework\App\FrontController\Interceptor->Magento\Framework\Interception\{closure}(Object(Magento\Framework\App\Request\Http))
#3 lib/internal/Magento/Framework/Interception/Interceptor.php(135): Magento\Store\App\FrontController\Plugin\RequestPreprocessor->aroundDispatch(Object(Magento\Framework\App\FrontController\Interceptor), Object(Closure), Object(Magento\Framework\App\Request\Http))
#4 app/code/Magento/PageCache/Model/App/FrontController/BuiltinPlugin.php(73): Magento\Framework\App\FrontController\Interceptor->Magento\Framework\Interception\{closure}(Object(Magento\Framework\App\Request\Http))
#5 lib/internal/Magento/Framework/Interception/Interceptor.php(135): Magento\PageCache\Model\App\FrontController\BuiltinPlugin->aroundDispatch(Object(Magento\Framework\App\FrontController\Interceptor), Object(Closure), Object(Magento\Framework\App\Request\Http))
#6 lib/internal/Magento/Framework/Interception/Interceptor.php(153): Magento\Framework\App\FrontController\Interceptor->Magento\Framework\Interception\{closure}(Object(Magento\Framework\App\Request\Http))
#7 generated/code/Magento/Framework/App/FrontController/Interceptor.php(26): Magento\Framework\App\FrontController\Interceptor->___callPlugins('dispatch', Array, NULL)
#8 lib/internal/Magento/Framework/App/Http.php(135): Magento\Framework\App\FrontController\Interceptor->dispatch(Object(Magento\Framework\App\Request\Http))
#9 lib/internal/Magento/Framework/App/Bootstrap.php(256): Magento\Framework\App\Http->launch()
#10 pub/index.php(37): Magento\Framework\App\Bootstrap->run(Object(Magento\Framework\App\Http))
#11 {main}
```

**Comment by dmitrii-fediuk** [comment]
```
foreach ($this->_routerList as $router) {
```
https://github.com/magento/magento2/blob/bce891d2/lib/internal/Magento/Framework/App/FrontController.php#L48

![02](https://user-images.githubusercontent.com/83204/34459793-70a122ac-ee24-11e7-95c4-f755dd9cb4ec.png)

**Comment by dmitrii-fediuk** [comment]
[The base router (`Magento\Framework\App\Router\Base`) ordering has been changed from **20** to **30** in **Magento 2.2.2** at **2017-10-15**](https://mage2.pro/t/topic/5241).

**Comment by dmitrii-fediuk** [comment]
Fixed in [0.4.5](https://github.com/mage2pro/portal/releases/tag/0.4.5).
